### PR TITLE
Fix Bug 1130160 - Extra '#' in section headers for roll up pages

### PR DESCRIPTION
**Pull request by kyoshino**
I pushed -f the branch before committing and #2710 was automatically closed :confused: I cannot reopen the PR so let me try again... :abillings agreed in the [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1130160) to use hidden anchors I proposed, rather than removing those # signs entirely.

### CODE REVIEW

**Comment by alexgibson** [comment]
Thanks @kyoshino looks good.

Only nit: it would be nice if the anchor links we're still keyboard accessible. Maybe we could make the heading text a link itself, the just hide/show the `#` symbol as a CSS pseudo selector on both `:hover` and `:focus`. Unless you have another idea? :smile:

**Comment by kyoshino** [comment]
:+1: to make the anchor accessible. I'll fix it tonight.

**Comment by kyoshino** [comment]
Updated!

**Comment by alexgibson** [comment]
This works great, thanks @kyoshino r+
